test/config_test.py

```diff
@@ -209,6 +211,10 @@ def test_signing_auth_header_not_set_when_calling_non_api_root_endpoint(self):
         )
         prepared_get = connection.session.prepare_request(get_request)
         self.assertFalse("authorization" in prepared_get.headers)
+        self.assertTrue(
+            prepared_get.headers["x-organization-id"] == connection.organization_id
+        )
+        self.assertTrue(prepared_get.headers["x-user-email"] == connection.email)
```

Comment:
nit: the `headers` method is case insensitive, but it would be nice if we could standardize on the casing. Would slightly prefer the original capitalization, i.e. `X-Organization-Id`